distributed/scheduler.py

```diff
@@ -5452,7 +5475,10 @@ def handle_long_running(self, key=None, worker=None, compute_duration=None):
         occ: double = ws._processing[ts]
         ws._occupancy -= occ
         parent._total_occupancy -= occ
+        # Cannot remove from processing since we're using this for things like
+        # idleness detection
```

Comment:
Wouldn't it be reasonable to consider a worker idle if it's only running seceded tasks?

Reply:
yes and no. Idleness is also used to detect whether workers should be downscaled which is a bad idea for jobs with long running tasks

Reply:
Yes great point. Maybe add that note to this comment?